### PR TITLE
fix(pop-confirm): [pop-confirm]  remove m funtion from  pc template

### PR DESCRIPTION
**Pull request by shenjunjian**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information
去掉 pc模板中的 m函数

## Summary by CodeRabbit

- **Improvements**
	- Enhanced the way CSS classes are applied to the popconfirm component, improving readability and maintainability.
	- The updated method for defining class names allows for better separation and handling of multiple classes.
	- Simplified class name construction to prevent formatting issues when custom classes are not provided.

### CODE REVIEW

File: packages/vue/src/popconfirm/src/pc.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
         trigger="manual"
         :width="width"
         :title="title"
-        :popper-class="m('tiny-popconfirm-popover', customClass)"
+        :popper-class="['tiny-popconfirm-popover', customClass]"
```
Comment (gimmyhehe) [inline packages/vue/src/popconfirm/src/pc.vue]: image
popper-class 是个string吧，这里没有没有像vue class一样会自动合并的功能

File: packages/vue/src/popconfirm/src/pc.vue [hunk 2]
```diff
@@ -8,7 +8,7 @@
         trigger="manual"
         :width="width"
         :title="title"
-        :popper-class="m('tiny-popconfirm-popover', customClass)"
+        :popper-class="'tiny-popconfirm-popover ' + customClass"
```
Comment (zzcr) [inline packages/vue/src/popconfirm/src/pc.vue]: 用户如果不传custom会出现'tiny-popconfirm-popover undefined'

File: packages/vue/src/popconfirm/src/pc.vue (hunk 1 above)
Comment (shenjunjian) [reply packages/vue/src/popconfirm/src/pc.vue]: 已修复

File: packages/vue/src/popconfirm/src/pc.vue (hunk 2 above)
Comment (shenjunjian) [reply packages/vue/src/popconfirm/src/pc.vue]: 已修复

**Review by zzcr — APPROVED** [review]

**Pull request merged**